<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-PRTCFI5V" failures="3" tests="6" name="BankingApp Test1" time="137.178" errors="0" timestamp="2023-01-17T20:16:17 IST">
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="10.736"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="9.418">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="22.756">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="28.143"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="27.189">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_AddNewCustomerTest_003" name="addNewCustomer" time="14.977"/>
</testsuite> <!-- BankingApp Test1 -->
